queries/just/injections.scm

```diff
@@ -0,0 +1,41 @@
+; The right side of =~ literals
+(regex_literal
+  (_) @injection.content
+  (#set! injection.language "regex"))
+
+; Default to bash highlighting for non-shebang recipes and commands
+(recipe
+  (recipe_body
+    !shebang) @injection.content
```

Comment:
This does not work. From what I can see shebang is a node, not a field. That syntax is field-only

Reply:
I did add a named field to support this upstream before suggesting it, https://github.com/IndianBoy42/tree-sitter-just/pull/151. I believe @Hubro updated to something more recent than that PR already

Reply:
Sou. That'll work then